frontend/src/Components/Api/SampleCounties.js

```diff
@@ -0,0 +1,297 @@
+// TODO: backend
+
+// function (async/await)
+// call the function and return the result
+
+export const counties = {
+    "del norte": {
+        total_energy: Math.random(),
+        total_session: Math.random(),
+        peak_energy: Math.random(),
+    },
```

Comment:
DRY up the object creation process here. No need to repeat all of this.

Reply:
This file isn't used anywhere anymore, so I will delete it.